### ISSUE TITLE
Codenameone Initializr does not prepare Eclipse project properly

### ISSUE THREAD

**Issue by ebardet** [codenameone/CodenameOne#3724]
After importing an Eclipse project prepared by the Initializr into the Eclipse IDE, Eclipse reports a java error in the ready to use application controller class.
![image](https://github.com/codenameone/CodenameOne/assets/6052513/62244af9-dc25-406a-995e-b6d576063722)

To work out this, I found there's a couple of extra manual steps:
1. Adding org.eclipse.m2e.apt.mode=jdt_apt to the ./common/.settings/org.eclipse.m2e.apt.prefs
![image](https://github.com/codenameone/CodenameOne/assets/6052513/23719e4a-57cd-4f3c-846b-5e0b01b1a6f4)

2. Adding target/generated-sources/rad-views to the .classpath
![image](https://github.com/codenameone/CodenameOne/assets/6052513/eaf9a48e-3adf-4d63-9851-28881a442b61)

These could be generated by the Initializr.

**Comment by ebardet** [comment]
All in the common directory.

Here is the .classpath (mind the JRE that is specific to my env)
[classpath.txt](https://github.com/codenameone/CodenameOne/files/12310772/classpath.txt)
In the common/.settings directory, org.eclipse.jdt.apt.core.prefs and org.eclipse.m2e.apt.prefs:
[prefs.zip](https://github.com/codenameone/CodenameOne/files/12310828/prefs.zip)

I don't think the other .prefs files are needed but if so, just ask.

**Comment by shannah** [comment]
Thanks for sharing.
This is a bit involved to get this to automatically generate this in the initializr.  As a half-measure, I've added a note to the documentation for CodeRAD:

https://shannah.github.io/CodeRAD/manual/#getting-started

![image](https://github.com/codenameone/CodenameOne/assets/2677562/03c266e3-cc6f-427b-9063-9995cff9d039)

I have added the same note to the default project readme for maven projects in the archetype, which will start appearing in projects starting next release.

Really appreciate you sharing as this will likely help many other users get started with CN1.
